packages/web/app/routes/feed[.xml].tsx

```diff
@@ -0,0 +1,41 @@
+import type { LoaderFunction } from "@remix-run/node";
+import { Feed } from "feed";
+import sortOn from "sort-on";
+import { useAwesomeYasunori } from "~/hooks/use-awsome-yasunori";
+
+const domain = "https://awesome.yasunori.dev";
+
+export const loader = (async () => {
+  const awsomeYasunori = await useAwesomeYasunori();
+
+  const feed = new Feed({
+    title: "Awesome Yasunori",
+    description: "Awesome Yasunori",
+    id: domain,
+    link: domain,
+    language: "ja",
+    copyright: "Nirunari Yasunori Sukinishiro License 2024",
+    feedLinks: {
+      rss: `${domain}/rss.xml`,
+    },
+  });
+
+  if (awsomeYasunori) {
```

Comment:
存在しないときは early return で 404 を返すテクニックです。

```suggestion
  if (!awsomeYasunori) {
    throw new Response(null, { status: 404 });
  }
```

Reply:
404なるほど！っ